tee-worker/ts-tests/integration-tests/common/utils/assertion.ts

```diff
@@ -38,7 +37,7 @@ export async function assertFailedEvent(
     type EventLike = Parameters<typeof isFailed>[0];
     const ievents: EventLike[] = events.map(({ event }) => event);
     const failedEvent = ievents.filter(isFailed);
-    /* 
+    /*
```

Comment:
Yes, it'd be great, but first we need to fix typescript to pick the right types. Same issue as: https://linear.app/litentry/issue/P-508/avoid-as-unknown-as-when-possible

Otherwise, the type guard on `.is(event)` won't work.

It took us some time to get that right on IDH 😄

Reply:
We're using is(event) approach in [di evm tests](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/ts-tests/integration-tests/di_evm_identity.test.ts#L166). So I feel like `as unknown as` might be being overused. I'll double-check when I start this task.